internal/db/sql/mysql/0001_init.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS store (
+  `key` VARCHAR(512) NOT NULL,
```

Comment:
`key` is a reserved word in MySQL, so we need to escape it. I considered renaming it, but it's not that big of a deal. If anyone has any strong opinions, we can absolutely rename it to avoid the escaping.